Dockerfile

```diff
@@ -32,7 +30,35 @@ RUN ln -sfT /dev/stderr /var/log/nginx/error.log; \
 	ln -sfT /dev/stdout /var/log/nginx/access.log; \
 	chown -R --no-dereference www-data:adm /var/log/nginx/
 
-COPY --from=curlimpersonate /usr/local/lib/libcurl-impersonate-ff.so /usr/local/lib/curl-impersonate/
+# install curl-impersonate library
+RUN set -xe \
+    && apt-get update \
+    && apt-get install --assume-yes curl \
+    && curlimpersonate_version=0.6.0 \
+    && { \
+        { \
+            [ $(arch) = 'aarch64' ] \
+            && archive="libcurl-impersonate-v${curlimpersonate_version}.aarch64-linux-gnu.tar.gz" \
+            && sha512sum="d04b1eabe71f3af06aa1ce99b39a49c5e1d33b636acedcd9fad163bc58156af5c3eb3f75aa706f335515791f7b9c7a6c40ffdfa47430796483ecef929abd905d" \
+        ; } \
+        || { \
+            [ $(arch) = 'armv7l' ] \
```

Comment:
I am not sure: would this binary also work on arm64 (e.g. in the case of Raspberry Pi)?

Reply:
I'm not sure what you mean. armv7l is arm32 / aarch32 and arm64 is aarch64. Actually, I run rss-bridge on a raspberry pi 4 and use the aarch64 library of curl-impersonate.

Reply:
My bad, `aarch64` is already in the list. My question was whether some of those strings were too specific and could also work for other values. For instance (I am not sure): `armv8l` vs. `aarch64`

Reply:
Might not be important for now, though

Reply:
The [rss-bridge image](https://hub.docker.com/r/rssbridge/rss-bridge/tags) is available for architectures:

- `linux/amd64` = `x86_64`
- `linux/arm/v7` = `armv7l`
- `linux/arm64` = `aarch64`

For new architectures, the build step will fail and an existing condition must be adjusted (if applicable) or a new condition must be added.